assets/js/admin/course-pricing-promo-sidebar.js

```diff
@@ -0,0 +1,60 @@
+import { __ } from '@wordpress/i18n';
+import { applyFilters, addFilter } from '@wordpress/hooks';
+import { PluginDocumentSettingPanel } from '@wordpress/edit-post';
+import { escapeHTML } from '@wordpress/escape-html';
+import { ExternalLink } from '@wordpress/components';
+
+/**
+ * Course Pricing Promo Sidebar component.
+ */
+
+const CoursePricingPromoSidebar = () => {
+	/**
+	 * Filters to get description for pricing component.
+	 *
+	 * @since 4.1.0
+	 *
+	 * @hook  senseiCoursePricingDescription
+	 * @return {Object}
+	 */
+	const description = applyFilters( 'senseiCoursePricingDescription', {
+		text: __(
+			'Sell this course using WooCommerce - integrates with subscriptions, memberships, affiliates, and more.'
+		),
+		url:
+			'https://senseilms.com/pricing/?utm_source=plugin_sensei&utm_medium=upsell&utm_campaign=course_pricing',
+	} );
+
+	return (
+		<PluginDocumentSettingPanel
+			name="sensei-course-theme"
+			title={ <>{ __( 'Pricing ', 'sensei-lms' ) }</> }
```

Comment:
Is the fragment needed?

Reply:
Same, [Link](https://github.com/Automattic/sensei/pull/4810/files#diff-0cdf94ac1cfb0a1e90adaef288fdb975f07f4c39ff34e0de2003256181b3ec93R30)